packages/core/tests/controllers/ApiController.test.ts

```diff
@@ -383,6 +383,64 @@ describe('ApiController', () => {
     expect(ApiController.state.search).toEqual(data)
   })
 
+  // Wallet search with whitespace and multiple words
+  it('should search wallet with search term', async () => {
+    const includeWalletIds = ['12341', '12342']
+    const excludeWalletIds = ['12343']
+    let data = [
+      {
+        id: '12341',
+        name: 'MetaMask',
+        image_id: '12341'
+      }
+    ]
+    OptionsController.setIncludeWalletIds(includeWalletIds)
+    OptionsController.setExcludeWalletIds(excludeWalletIds)
+
+    let fetchSpy = vi.spyOn(api, 'get').mockResolvedValue({ data })
+    const fetchImageSpy = vi.spyOn(ApiController, '_fetchWalletImage').mockResolvedValue()
+
+    // Whitespace
+    await ApiController.searchWallet({ search: 'MetaMask    ' })
```

Comment:
Test with leading whitespace not just trailing would be helpful too

Reply:
@chris13524 Done!